app/src/main/java/com/framgia/englishconversation/screen/imageDetail/ImageDetailViewModel.java

```diff
@@ -23,21 +28,23 @@
     private FragmentManager mManager;
     private Fragment mFragment;
     private Context mContext;
+    private Navigator mNavigator;
     private OnMediaModelItemTouchListener<List<MediaModel>> mListener =
             new OnMediaModelItemTouchListener<List<MediaModel>>() {
-            @Override
-            public void onTouchListener(List<MediaModel> items, int postion) {
-                mContext.startActivity(
-                    SelectedImageDetailActivity.getIntent(mContext, items, postion));
-            }
-        };
+                @Override
```

Comment:
ERROR: 'method def modifier' have incorrect indentation level 16, expected level should be one of the following: 8, 12.